File: drestaurant-libs/drestaurant-common/src/main/java/com/drestaurant/order/domain/model/OrderState.java
package com.drestaurant.order.domain.model;

/**
 * @author: idugalic
 */
public enum OrderState {
	CREATE_PENDING,
	VERIFIED_BY_CUSTOMER,
	VERIFIED_BY_RESTAURANT,
	PREPARED,
	READY_FOR_DELIVERY,
	DELIVERED,
	REJECTED,
	CANCEL_PENDING,
	CANCELLED
}

File: drestaurant-libs/drestaurant-common/src/main/kotlin/com/drestaurant/order/domain/model/OrderState.kt
package com.drestaurant.order.domain.model

/**
 * @author: idugalic
 */
enum class OrderState {
    CREATE_PENDING,
    VERIFIED_BY_CUSTOMER,
    VERIFIED_BY_RESTAURANT,
    PREPARED,
    READY_FOR_DELIVERY,
    DELIVERED,
    REJECTED,
    CANCEL_PENDING,
    CANCELLED
}